src/richie/apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -83,6 +83,7 @@ <h1 class="subheader__title" property="name">{% render_model current_page "title
                                 {% if instance.duration or current_page.publisher_is_draft %}
                                 <li class="characteristics__item">
                                     <svg role="img" class="characteristics__icon" aria-hidden="true">
+                                        <title>{% trans "Duration" %}</title>
```

Comment:
I'm not sure the 3 `<svg>` changed on this page should still have an `aria-hidden` attribute? It seems they convey meaning.

Reply:
Indeed. This is probably a mistake. If they have a `title`, they should be read by the screen reader. Will fix this.

Reply:
Sorry about the wrong advice, now that i'm working on it and watched the code around this, the `aria-hidden` were actually the right choice. Because Duration/Effort/Pace are all written in plain text just after, so it's actually a bad idea to repeat it in the icon.